# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/frame_builder.rs
# Commit: 2b42abbdb0df
# Full Hash: 2b42abbdb0df38f31dfa1178fe3b5f773f8e4812
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 9fa6f71111d0 -r 2b42abbdb0df gfx/wr/webrender/src/frame_builder.rs
--- a/gfx/wr/webrender/src/frame_builder.rs	Sun Feb 20 18:56:54 2022 +0000
+++ b/gfx/wr/webrender/src/frame_builder.rs	Sun Feb 20 18:56:55 2022 +0000
@@ -288,8 +288,6 @@
 pub struct PictureState {
     pub map_local_to_pic: SpaceMapper<LayoutPixel, PicturePixel>,
     pub map_pic_to_world: SpaceMapper<PicturePixel, WorldPixel>,
-    pub map_pic_to_raster: SpaceMapper<PicturePixel, RasterPixel>,
-    pub map_raster_to_world: SpaceMapper<RasterPixel, WorldPixel>,
 }
 
 impl FrameBuilder {